src/sentry/types/region.py

```diff
@@ -167,6 +167,7 @@ def get_region_for_organization(organization: Organization) -> Region:
     # Backend representation to be determined. If you are working on code
     # that depends on this method, you can mock it out in unit tests or
     # temporarily hard-code a placeholder.
+    return get_region_by_name("region")
     raise NotImplementedError
```

Comment:
It looks like this exception can be removed
```suggestion
```

Reply:
oh oops I didn't meant to check this file, I only do this for testing locally. Good eye though :+1: